plugins/must-gather/skills/must-gather-analyzer/SKILL.md

```
@@ -189,6 +189,33 @@ Shows monitoring information:
+Interactive web-based analysis for cluster infrastructure and autoscaler:
+- Examines cluster autoscaler configuration and behavior
+- Reviews cluster operators status and health
+- Analyzes Machines, MachineSets, and Nodes configuration
+- Inspects Control Plane MachineSets
+- Checks Machine API operator status
+- Reviews Machine Config operator status
+- Examines Cloud Controller Manager details
+- Provides visual interface for investigating autoscaler scaling decisions
+- Helps debug autoscaler and cluster infrastructure issues
+- Automatically handles permissions and browser launching
+- Runs on http://127.0.0.1:8080
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)